examples/lead-agent/backend/app.py

```
@@ -0,0 +1,147 @@
+def health_check():
+    """Health check endpoint"""
+    return jsonify({
+        'status': 'healthy',
+        'service': 'lead-score-api',
+        'version': '1.0.0'
+    }), 200
+
+
+if __name__ == '__main__':
+    port = int(os.getenv('PORT', 8000))
+    app.run(host='0.0.0.0', port=port, debug=True)
```

Ruff (0.14.2)

Possible binding to all interfaces

(S104)